src/models.py

```diff
@@ -26,3 +28,13 @@ def set_password(self, password: str) -> None:
     def check_password(self, password_to_check: str) -> bool:
         """Use this method to check User's password."""
         return self._get_password_hash(password_to_check) == self.password_hash
+
+
+class UserSession(Base):  # pylint: disable=too-few-public-methods
+    """A model class for User_session table."""
+
+    __tablename__ = "user_session"
+
+    id = Column(Integer, primary_key=True)  # noqa: A003
+    session_id = Column(String, nullable=False, unique=True)
+    user_id = Column(Integer, ForeignKey("users.id"))
```

Comment:
Seems we should add `nullable=False` constraint here, because every existing session must have a user associated with this session.

When you add this constraint you will need to downgrade database by one, remove last migration, create new migration and upgrade db.